code/modules/shuttle/supply.dm

```diff
@@ -662,6 +662,13 @@
 			investigate_log("[key_name(usr)] has sent the supply shuttle away. Remaining points: [shuttle_master.points]. Shuttle contents: [shuttle_master.sold_atoms]", "cargo")
 		else if(!shuttle_master.supply.request(shuttle_master.getDock("supply_home")))
 			post_signal("supply")
+			if(prob(10))
```

Comment:
Make this `if(shuttle_master.shoppinglist.len && prob(10))` so it will only include extra crates if they actually ordered crates. Otherwise you could sit there and request the shuttle empty for free crates which you can send back for free points. Gotta spend money to make money, right?